src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -306,8 +327,13 @@ ApiGlobe.prototype.setRealisticLightingOn = function(value) {
 ApiGlobe.prototype.setLayerVisibility = function(id, visible) {
 
     this.scene.getMap().setLayerVisibility(id, visible);
-
+    eventLayerChanged.layerId = id;
+    eventLayerChangedVisible.layerId = id;
+    //eventLayerChanged.initCustomEvent("layerchanged:visible", false, false, false);
```

Comment:
Called when the visibility of a layer has changed, return an object with two properties, id and visible